app/jobs/risc_delivery_job.rb

```diff
@@ -14,10 +14,13 @@ class RiscDeliveryJob < ApplicationJob
     *NETWORK_ERRORS,
     wait: :polynomially_longer,
     attempts: 2,
-  )
+  ) do |_job, _exception|
+  end
```

Comment:
It's a little unclear from this why we have a blank block, I think this would be a good candidate for a comment, or since we have 2, extracing a proc into a variable with a clear name that indicates what's going on

```ruby
  FINAL_EXCEPTION_HANDLER_NOOP = proc { |_job, _exception| }.freeze

  retry_on(
    *NETWORK_ERRORS,
    wait: :polynomially_longer,
    attempts: 2,
    &FINAL_EXCEPTION_HANDLER_NOOP
  )
```

Reply:
To me, a comment would be clear and concise, but if a named `Proc` would be idiomatic, I'll go with that.

Reply:
It's 6 of one, half a dozen of the other. I don't think one is more idiomatic